lib/src/models/movie_model.dart

```diff
@@ -0,0 +1,29 @@
+import 'result_movie_model.dart';
+
+class MovieModel {
+  late int _page;
+  late int _totalResults;
+  late int _totalPages;
+  List<ResultMovie> _results = [];
+
+  MovieModel.fromJson(Map<String, dynamic> parsedJson) {
```

Comment:
When you create this type of constructors we can use the word `factory` at the begging of the method declaration

Reply:
Solved